doc/source/operations/upgrading-openstack.rst

```diff
@@ -35,11 +35,21 @@ Notable changes in the |current_release| Release
 There are many changes in the OpenStack |current_release| release described in
 the release notes for each project. Here are some notable ones.
 
-RabbitMQ 4.0
+RabbitMQ 4.1
 ------------
 
-RabbitMQ is being upgraded to 4.0 in Epoxy. Existing transient queues must be
-migrated on Caracal prior to upgrading.
+.. warning::
+
+   This section must be followed prior to upgrading to OpenStack Epoxy.
+
+From Epoxy release, OpenStack requires RabbitMQ 4.0 or 4.1.
+StackHPC Kayobe Config sets RabbitMQ 4.1 as a default for Epoxy release.
+RabbitMQ 4.0 or greater requires durable queues.
+Existing transient queues must be migrated to durable queues with Queue Manager.
+RabbitMQ 3.13 needs to be upgraded to 4.1.
```

Comment:
I think it would be best to shorten this just down to the main points. The person doing the upgrade doesn't need to know _why_ the changes are made, they just need to know the instructions for how to do it. 

```suggestion
StackHPC Kayobe Config sets RabbitMQ 4.1 as the default for the Epoxy release.
Existing transient queues must be migrated to durable queues with Queue Manager before upgrading to RabbitMQ 4.1.
```